product_docs/docs/migration_portal/4.0/index.mdx

```diff
@@ -1,5 +1,5 @@
 ---
-title: "EDB Postgres Migration Portal"
+title: "Migration Portal"
 directoryDefaults:
   description: "EDB Postgres Migration Portal Version 3.2.0 Documentation and release notes. EDB Postgre Migration Portal converts Oracle Schema to PostgreSQL online. "
```

Comment:
@drothery-edb @moiznalwalla Does this information show up somewhere or is just meta data?  Note that the description says "EDB Postgres Migration Portal Version 3.2.0".  Should it be updated to say "EDB Postgres Migration Portal Version 4.0.0"?

Reply:
It can show up. Let's change it to a short description of the product something along the lines of:
  description: "Migration Portal converts Oracle schema to Postgres online.

Reply:
Replaced all occurrences of "EDB Postgres Migration Portal" to "Migration Portal" and updated the version no to 4.0